core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -727,8 +769,8 @@ private[spark] class AppStatusListener(
     }
   }
 
-  private def update(entity: LiveEntity, now: Long): Unit = {
-    entity.write(kvstore, now)
+  private def update(entity: LiveEntity, now: Long, last: Boolean = false): Unit = {
```

Comment:
nit: maybe change `last` to `isLast` ?

Reply:
I prefer how the current version reads on the call site, e.g.:

    update(exec, now, last = true)

Also, Spark generally avoids Java-beans-style prefixes in Scala code (like "is" or "get").

Reply:
maybe `checkTriggers` is better than `last`?